src/match/serializers.py

```diff
@@ -0,0 +1,137 @@
+import json
+
+from location.serializers import LocationSerializer
+from match.models import Match
+from person.simple_serializers import SimpleUserSerializer
+from rest_framework import serializers
+
+
+class MatchSerializer(serializers.ModelSerializer):
+    matched_user = serializers.SerializerMethodField("get_matched_user")
+    accepted_ids = serializers.SerializerMethodField("get_accepted_ids")
+    proposed_meeting_times = serializers.SerializerMethodField(
+        "get_proposed_meeting_times"
+    )
+    proposed_locations = serializers.SerializerMethodField("get_proposed_locations")
+    meeting_location = serializers.SerializerMethodField("get_meeting_location")
+
+    def __init__(self, *args, **kwargs):
+        self.request_user = kwargs.pop("user")
+        super().__init__(*args, **kwargs)
+
+    def get_matched_user(self, match):
+        if self.request_user == match.user_1:
+            return SimpleUserSerializer(match.user_2).data
+        elif self.request_user.id == match.user_2.id:
+            return SimpleUserSerializer(match.user_1).data
+
+    def get_accepted_ids(self, match):
+        if match.accepted_ids is None:
+            return None
+        accepted_ids = json.loads(match.accepted_ids)
+        if accepted_ids == []:
+            return None
+        return accepted_ids
```

Comment:
can't you do `return accepted_ids or None`? you should test this out in the python interpreter first, or you can enter the shell with `python manage.py shell`